src/tactics/LocalTransport.js

```diff
@@ -27,14 +27,14 @@ export default class LocalTransport {
    */
   static createGame(gameData) {
     let transport = new LocalTransport();
-    transport._post({ type:'create', data:gameData });
+    transport._post({ type:'create', data:gameData.data });
```

Comment:
@latin-programmer Once you stop nesting the data, you can revert this change as well.